frontend/src/components/elements/DeckGlChart/DeckGlChart.tsx

```diff
@@ -15,40 +15,23 @@
  * limitations under the License.
  */
 
-import React, { PureComponent } from "react"
-import DeckGL, {
-  ArcLayer,
-  GridLayer,
-  HexagonLayer,
-  LineLayer,
-  PointCloudLayer,
-  ScatterplotLayer,
-  ScreenGridLayer,
-  TextLayer,
-  // @ts-ignore
-} from "deck.gl"
+import React, { ReactElement, useEffect, useState } from "react"
 import Immutable from "immutable"
 import { StaticMap } from "react-map-gl"
-import { dataFrameToArrayOfDicts } from "lib/dataFrameProto"
 import withFullScreenWrapper from "hocs/withFullScreenWrapper"
 import withMapboxToken from "hocs/withMapboxToken/withMapboxToken"
 import "mapbox-gl/dist/mapbox-gl.css"
 import "./DeckGlChart.scss"
+import { buildLayer, getStyleUrl } from "./DeckGlChartUtil"
+import DeckGL from "deck.gl"
```

Comment:
This errors in eslint for this being below a relative input. Move it up with the absolute imports.